packages/shared/src/hdf5-utils.ts

```diff
@@ -33,7 +35,7 @@ export function buildEntityPath(
 /* ----- TYPES ----- */
 
 export function intType(
-  size: 8 | 16 | 32 | 64 = 32,
+  size = 32,
```

Comment:
Had to loosen this up to allow creating numeric types from computed sizes - i.e. `length * 8`.